gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -237,6 +242,14 @@ protected void createButtonsForButtonBar(Composite parent) {
       traceInput.addModifyListener(modifyListener);
 
       modifyListener.handleEvent(null); // Set initial state of widgets.
+
+      traceInput.validationStatus.addListener(new PropertyChangeListener() {
+
+        @Override
+        public void propertyChange(PropertyChangeEvent event) {
+          ok.setEnabled(traceInput.isReady());
```

Comment:
There should be an command override.  Please do **not** submit this until override is implemented.  We are working with the vendors to ensure integration works.  Adding validation is going to slow down the progress.

Reply:
Yeah. I'm just starting with the override. Will update the PR with it

Reply:
Regarding the Pixel failure, try to run `bazel run gapit validate_gpu_profiling` and confirm if it passes there. If it fails there, then the device failed validation indeed. Currently, there is some textures/fragment that is a bit flaky and Peiyong is working on them.

Reply:
Added the commandline override. The help menu will show the command as well.